components/tooltip/test/tooltip-help.visual-diff.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
+	<script type="module">
+		import '../../../test/typography-preload.js';
+		import '../tooltip-help.js';
+		import { forceFocusVisible } from '../../../helpers/focus.js';
+		window.forceFocusVisible = forceFocusVisible;
+	</script>
+	<title>d2l-tooltip-help</title>
+	<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1.0">
+	<meta charset="UTF-8">
+	<style>
+		html, body {
+			box-sizing: border-box;
+			height: 100%;
+			margin: 0;
+			width: 400px;
+		}
+		p {
+			height: 200px;
+			padding: 20px;
+			text-align: left;
+		}
+		p d2l-tooltip-help {
+			display: inline;
+		}
+		.visual-diff {
+			height: 100%;
+			margin: 0;
+			position: relative;
+			width: 100%;
+		}
+		.visual-diff > d2l-tooltip-help {
+			display: block;
+			margin-bottom: 10rem;
+			padding: 20px;
+		}
+	</style>
+</head>
+<body class="d2l-typography">
+	<div class="visual-diff">
```

Comment:
Ah ok, I was initially following the `tooltip` visual diff as an exemplar, but looks like that's another case where just wrapping all the test cases in one `visual-diff div` is the exception, rather than the convention.